openfl/display/Graphics.hx

```diff
@@ -244,7 +244,7 @@ import js.html.CanvasRenderingContext2D;
 	 */
 	public function beginGradientFill (type:GradientType, colors:Array<Dynamic>, alphas:Array<Dynamic>, ratios:Array<Dynamic>, matrix:Matrix = null, spreadMethod:Null<SpreadMethod> = null, interpolationMethod:Null<InterpolationMethod> = null, focalPointRatio:Null<Float> = null):Void {
 		
-		__commands.push (BeginGradientFill (type, colors, alphas, ratios, matrix, spreadMethod, interpolationMethod, focalPointRatio));
+		__commands.writeBeginGradientFill (type, cast colors, cast alphas, cast ratios, matrix, spreadMethod, interpolationMethod, focalPointRatio);
```

Comment:
These casts don't seem safe, and a little surprised that it would build.

Since the public API takes Array<Dynamic>, I don't think we'll be adding any allocs by passing them as Array<Dynamic> all the way through.